test/Microsoft.Health.Fhir.Shared.Tests.Integration/Features/ChangeFeed/SqlServerFhirResourceChangeCaptureTests.cs

```diff
@@ -96,56 +74,34 @@ public async Task GivenADatabaseSupportsResourceChangeCapture_WhenInsertingAReso
-                        var resourceChanges = await resourceChangeDataStore.GetRecordsAsync(1, 200, CancellationToken.None);
-
-                        Assert.NotNull(resourceChanges);
-                        Assert.Equal(2, resourceChanges.Count);
-
-                        var resourceChangeData = resourceChanges.Where(x => x.ResourceVersion == 2).First();
-
-                        Assert.Equal(deserialized.Id, resourceChangeData.ResourceId);
-                        Assert.Equal(1, resourceChangeData.ResourceChangeTypeId); // 1 is for resource udpate.
-                    }
-                    catch (Exception e)
-                    {
-                        throw new InvalidOperationException($"Failure using schema version {i}", e);
-                    }
-                    finally
-                    {
-                        await fhirStorageTestsFixture?.DisposeAsync();
-                    }
-                }
+                // add a new resource
+                var saveResult = await _fixture.Mediator.UpsertResourceAsync(Samples.GetJsonSample("Weight"));
+
+                // update the resource
+                var newResourceValues = Samples.GetJsonSample("WeightInGrams").ToPoco();
+                newResourceValues.Id = saveResult.RawResourceElement.Id;
+
+                // save updated resource
+                var updateResult = await _fixture.Mediator.UpsertResourceAsync(newResourceValues.ToResourceElement(), WeakETag.FromVersionId(saveResult.RawResourceElement.VersionId));
+                var deserialized = updateResult.RawResourceElement.ToResourceElement(Deserializers.ResourceDeserializer);
+
+                // get resource changes
+                var resourceChangeDataStore = new SqlServerFhirResourceChangeDataStore(GetSqlConnectionFactory(_fixture.DatabaseName), NullLogger<SqlServerFhirResourceChangeDataStore>.Instance);
+                var resourceChanges = await resourceChangeDataStore.GetRecordsAsync(1, 200, CancellationToken.None);
+
+                Assert.NotNull(resourceChanges);
+                Assert.Single(resourceChanges.Where(x => x.ResourceVersion == 2 && x.ResourceId == deserialized.Id));
+
+                var resourceChangeData = resourceChanges.Where(x => x.ResourceVersion == 2 && x.ResourceId == deserialized.Id).FirstOrDefault();
+
+                Assert.NotNull(resourceChangeData);
+                Assert.Equal(1, resourceChangeData.ResourceChangeTypeId); // 1 is for resource udpate.
             }
             catch (Exception e)
             {
-                throw new Exception($"An error occurred on a resource change capture integration test", e);
+                throw new Exception($"An error occurred on a resource change capture integration test. Failure using schema version {SchemaVersionConstants.Max}.", e);
             }
```

Comment:
Now that we aren't running this test for multiple schema versions, I don't think we need the try-catch block!

Reply:
Removed the try-catch blocks.